procrastinate/retry.py

```diff
@@ -46,24 +46,39 @@ def get_schedule_in(self, attempts: int) -> Optional[int]:
 @attr.dataclass(kw_only=True)
 class RetryStrategy(BaseRetryStrategy):
     """
-    The RetryStrategy class should handle simple retry strategies.
+    The RetryStrategy class should handle classic retry strategies.
+
+    You can mix and match several waiting strategies. The formula is::
+
+        total_wait = wait + lineal_wait * attempts + exponential_wait ** attempts
 
     Parameters
     ----------
     max_attempts:
         The maximum number of attempts the job should be retried
     wait:
-        The number of seconds to wait between attempts
+        Number of seconds to wait between attempts will follow a constant series
+        (e.g. if 3, then successive runs will wait 3, 3, 3, 3, 3 seconds)
```

Comment:
Les documentations de ces trois arguments ne sont pas claires du tout :o
Je propose ca, c'est un debut de mieux je trouve :
```
        The number of seconds used to compute the constant time to wait between retries
        (e.g. if 3, then successive runs will wait 3, 3, 3, 3, 3 seconds)
```
A generaliser sur les `linear_wait` et `exponential_wait`

Reply:
(don't forget we're in an open source repo :us: :gb:)

`The number of seconds used to compute the linear/exponential time to wait between retries` don't strike me as a really better solution, but I admit writing those lines was hard. I still think we could find a better solution.

Reply:
:new_zealand: 
The current sentence `Number of seconds to wait between attempts will follow a constant series` doesn't make sense to me :o We can ask an actual native for help :>

Reply:
:canada: 
Ah I said "series" instead of "sequence", sorry. Would that have helped, tabarnak ?

Reply:
New iteration?
```
Use this if you want to use a constant backoff. Give a number of seconds as argument, it will be used to compute the backoff: if 3, then successive runs will wait 3, 3, 3, 3, 3 seconds.
```

Reply:
Can I let you add the appropriate commits on top of this PR ? Thanks :)

Reply:
Done :+1:

Reply:
Thanks ❤️